parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -70,7 +70,7 @@ if [[ "${SHOULD_CONFIGURE_CUSTOM_CA_TRUST}" == "true" ]]; then
     configureCustomCaCertificate || $ERR_UPDATE_CA_CERTS
 fi
 
-{{GetOutboundCommand}}
+$OUTBOUND_COMMAND
```

Comment:
are we sure we don't need an "eval" here or something?

Reply:
waiting on agentbaker e2e for this one but afaict this will do the correct word splitting/evaluation

Reply:
hmm you're right, didn't quite work